seed.js

```diff
@@ -25,27 +25,14 @@ var sampleRestaurants = [{
   description: 'more creative',
 }];
 
-
 db.Restaurant.remove({}, function(err, restaurants){
   console.log('removed all restaurants');
 
-    db.Restaurant.create(sampleRestaurants, function(err, restaurants){
+  db.Restaurant.create(sampleRestaurants, function(err, restaurants){
     // code in here runs after all , restaurants are created
-    if (err) { return console.log('ERROR', err); }
+    if (err) { console.log('ERROR', err); }
```

Comment:
Having the `return` means it will return before every running the `console.log`.